### PR TITLE
Styling updates

### PR DESCRIPTION
**Pull request by kepea1**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,7 @@ node_modules
 
 # environment variables
 .env
-.env.production
\ No newline at end of file
+.env.production
+
+dist
+/dist
```
Comment (lpoulter1) [inline .gitignore]: ah sorry didn' realise you did this also. you can a conflict now. just undo this change and it should be good. let me know if you need help.

**Review by lpoulter1 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (kepea1) [reply .gitignore]: yer i saw you did it and wasnt sure of it would conflict or not ... i guess i know now!

**Comment by kepea1** [comment]
.
